### DISCUSSION TITLE
Can text-generation-webui do multi model workers?

### DISCUSSION THREAD

**Discussion by nbollman** [oobabooga/text-generation-webui#4289] [Q&A]
Im trying to recreate Microsoft's AutoGen with Local LLM [link](https://microsoft.github.io/autogen/blog/2023/07/14/Local-LLMs/#for-further-reading) but using the Text-Generation-Webui... however I cant find anything on possibly loading more than one model through Text-Generation-Webui, is this a fruitless endeavor?
How multiple models are loaded via FastChat:
```
python -m fastchat.serve.multi_model_worker \
    --model-path lmsys/vicuna-7b-v1.3 \
    --model-names vicuna-7b-v1.3 \
    --model-path chatglm2-6b \
    --model-names chatglm2-6b
```
Im trying to build my AutoGen project utilizing Mistral-7b models, and it appears my only option is the Transformers library with fastchat as GPTQ for Llama does not support Mistral... leaving my project rather slow...

--multimodal-pipeline sort of loads two models at once, seems like a missed opportunity to not be able to load multiple models for text-generation-webui... :(

**Comment by kasukanra** [comment]
I'm also interested in loading two or more models (i.e. `Mistral` and `LLaVA`) for something like the `api`. Currently, I think they have separate default hosts at `localhost:5001/v1` and `localhost:5000/api/v1/generate`, so I was also under the impression that it was sort of supported.
